ui-speedspacechart/src/components/helpers/drawElements/tickX.ts

```diff
@@ -59,7 +59,7 @@ export const drawTickX = ({ ctx, width, height, store }: DrawFunctionParams) =>
       const textPositionX = MARGIN_LEFT + xPosition * i + CURVE_MARGIN_SIDES / 2;
       let opacity = 1;
 
-      // low progressivily opacity for text when text is near borders or cursor, except for 0
+      // low progressively opacity for text when text is near borders or cursor, except for 0
```

Comment:
The spelling is now correct but the grammar still doesn't seem to be… Maybe the intent of the comment was "reduce progressively opacity [...]"?